client/pages/users-posts.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+export default class UsersPosts extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      posts: []
+    };
+  }
+
+  componentDidMount() {
+    fetch('/api/users-posts/1')
+      .then(res => res.json())
+      .then(posts => {
+        this.setState({ posts });
+      });
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <h3 className="heading my-sm-4">What you&apos;ve Shared with Other Earthlings </h3>
+            {
+              this.state.posts.map(post => {
+                return (
+                  <div key={post.postId}>
+                    <OnePost post={post} />
+                  </div>
+                );
+              })
+            }
+          </div>
+    );
+  }
+}
+
+function OnePost(props) {
+  const { title, image, username } = props.post;
+  let { tags } = props.post;
+  if (tags.length > 1) {
+    let tagsString = '';
+    for (let i = 0; i < tags.length; i++) {
+      if (i === tags.length - 1) {
+        tagsString = tagsString + ' ' + tags[i];
+      } else {
+        tagsString = tagsString + ' ' + tags[i] + ',';
+      }
+    }
+    tags = tagsString;
```

Comment:
You can use `Array.prototype.join()` here.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join